### PR TITLE
Adding codegen for Python using http.client

### PR DESCRIPTION
**Pull request by morpheus-30**

### CODE REVIEW

**Comment by morpheus-30** [comment]
Issue for this PR #65

**Comment by animator** [comment]
Hi @morpheus-30 
Can you kindly take a look at the codegen again as the generated codes are incorrect.

Screenshot 2023-06-25 at 2 44 03 PM

Screenshot 2023-06-25 at 2 43 01 PM

**Comment by morpheus-30** [comment]
I fixed the typo, kindly review :)

**Comment by animator** [comment]
Hi @morpheus-30, I reviewed the code and issues still persist.

Screenshot 2023-07-04 at 9 08 19 PM
Screenshot 2023-07-04 at 9 08 41 PM

.  

..

..

You have to also understand that test cases should be sacrosanct and you should run them all manually to verify.
Here is the error message after I ran an expected code written in the test case:

Screenshot 2023-07-04 at 9 10 17 PM

**Comment by morpheus-30** [comment]
Got that, I'll see to it

File: test/widget_test.dart
Comment (animator) [inline test/widget_test.dart:1]: Please remove this file from the pR.

**Pull request merged**